### PR TITLE
Release 0.0.7 and bump version

### PR DESCRIPTION
**Pull request by alpar-t**
This PR bumps the version after releasiong 0.0.7 but also makes some changes so fewer updates are required when this happens

### CODE REVIEW

**Review by jamietanna — COMMENTED** [review]
Can we maybe start using Semantic Versioning for this? Or if not "true" SemVer, at least version this as 0.1.0 and going forwards we can try and use minor/patch more appropriately?

**Comment by alpar-t** [comment]
> Can we maybe start using Semantic Versioning for this? Or if not "true" SemVer, at least version this as 0.1.0 and going forwards we can try and use minor/patch more appropriately?

Yes, we can move to `0.1.0` with the next release

File: version-next [hunk 1]
```diff
@@ -0,0 +1 @@
+0.0.9
```
Comment (jamietanna) [inline version-next:1]:
```suggestion
0.1.0
```

**Review by jamietanna — COMMENTED** [review]
Any docs needed i.e. a `RELEASING.md`?

File: version-next [hunk 2]
```diff
@@ -0,0 +1 @@
+0.1.0
```
Comment (jamietanna) [inline version-next:1]: Nit: is there any reason we don't have a newline at the end of the file? I.e. is it required to not have one?

(I usually recommend us adding newlines at end of file i.e. https://www.jvt.me/posts/2019/09/02/newlines-all-git-files/ so diff noise is reduced when different folks update them)

**Review by jamietanna — APPROVED** [review]

**Comment by jamietanna** [comment]
Re docs we've discussed in DM and we'll document it internally, not in this repo (aside from maybe sharing a link to the docs)

File: version-next (hunk 2 above)
Comment (alpar-t) [reply version-next:1]: Good point, I'll add it with the next update.

**Pull request merged**